plugins/Live/Model.php

```diff
@@ -176,25 +176,18 @@ public function queryEcommerceItemsForOrder($idVisit, $idOrder, $actionsLimit)
      * @param $period
      * @param $date
      * @param $segment
-     * @param $countVisitorsToFetch
+     * @param $limit
      * @param $visitorId
      * @param $minTimestamp
      * @param $filterSortOrder
      * @return array
      * @throws Exception
      */
-    public function queryLogVisits($idSite, $period, $date, $segment, $countVisitorsToFetch, $visitorId, $minTimestamp, $filterSortOrder)
+    public function queryLogVisits($idSite, $period, $date, $segment, $offset, $limit, $visitorId, $minTimestamp, $filterSortOrder)
     {
-        list($sql, $bind) = $this->makeLogVisitsQueryString($idSite, $period, $date, $segment, $countVisitorsToFetch, $visitorId, $minTimestamp, $filterSortOrder);
-
-        try {
-            $data = Db::fetchAll($sql, $bind);
-            return $data;
-        } catch (Exception $e) {
-            echo $e->getMessage();
-            exit;
```

Comment:
This `exit` was added in https://github.com/piwik/piwik/commit/977d0593a41db4b14f0490f726469400a762e13b#diff-11527d0f07d21b83277153a2ebb852acR306 but didn't really make sense to me. Maybe it was just the way we handled it back then